lib/KafkaBacklogMetrics.js

```diff
@@ -56,6 +56,25 @@ const taskProcessingTime = metrics.ZenkoMetrics.createHistogram({
     buckets: [0.01, 0.1, 1, 10, 50, 100, 200, 300],
 });
 
+const taskQueuedTime = metrics.ZenkoMetrics.createHistogram({
```

Comment:
is this metrics useful?
in 99% of cases, queues will be empty, and queue time will be 0... And in the other cases, the time will be equal to the duration of the processing.